loggly.go

```diff
@@ -110,7 +107,7 @@ func (c *Client) Send(msg Message) error {
 
 	c.buffer = append(c.buffer, json)
 
-	debug("buffer (%d/%d) %v", len(c.buffer), c.BufferSize, msg)
+	//debug("buffer (%d/%d) %v", len(c.buffer), c.BufferSize, msg)
```

Comment:
Rather than commenting out the `debug()` calls, I'd prefer we just remove them entirely.